doc/cli/vast-start.md

```diff
@@ -1,3 +1,23 @@
-The `start` command spins up a VAST node. Starting a node is the first step
-when deploying VAST as a continuously running server. The process runs in the
+The `start` command spins up a VAST node. Starting a node is the first step when
+deploying VAST as a continuously running server. The process runs in the
 foreground and uses standard error for logging. Standard output remains unused.
+
+By default, the `vast start` command creates a `vast.db` directory in the
+current working directory. It is recommended to set the options for the node in
+the `vast.conf` file, such that they are picked up by all client commands as
+well.
+
+The `node` (sometimes called `server` or `server process`) is the core piece of
```

Comment:
Actually, the node is not the same as the server. See [our docs](https://docs.tenzir.com/vast/architecture/system-structure) what I mean. We use "server" when talking about the entire `vast` process that runs via `vast start`. The "client" is the other process to access the server. The "node" is the main container actor inside the server that contains everything else.

Reply:
I suggest using the wording as in the referenced docs.